memgpt/server/server.py

```diff
@@ -1770,3 +1770,38 @@ def retry_agent_message(self, agent_id: str) -> List[Message]:
         # Get the current message
         memgpt_agent = self._get_or_load_agent(agent_id=agent_id)
         return memgpt_agent.retry_message()
+
+    # TODO(ethan) wire back to real method in future ORM PR
+    def get_current_user(self) -> User:
```

Comment:
@sarahwooders @norton120 

FYI this is the stub method I'm talking about in the OP